server/applicationPattern/applicationPattern/services/ElasticsearchService.js

```diff
@@ -290,7 +290,7 @@ async function getNodeAsync(uuid) {
   let address = await TcpClientInterface.getRemoteAddressAsync(tcpClient[0]);
   let port = await TcpClientInterface.getRemotePortAsync(tcpClient[0]);
   let remoteProtocol = await TcpClientInterface.getRemoteProtocolAsync(tcpClient[0]);
-  return remoteProtocol + "://" + address[onfAttributes.TCP_CLIENT.IP_ADDRESS][onfAttributes.TCP_CLIENT.IPV_4_ADDRESS] + ":" + port;
```

Comment:
As getRemoteProtocolAsync function now supports fetching of protocol directly instead of onf notation can we remove these changes

Reply:
Yes. Removing PR.